pkgs/development/compilers/llvm/3.9/clang/default.nix

```diff
@@ -13,7 +13,7 @@ let
       mv clang-tools-extra-* $sourceRoot/tools/extra
     '';
 
-    buildInputs = [ cmake libedit libxml2 llvm python ];
+    buildInputs = [ buildPackages.cmake libedit libxml2 llvm python ];
```

Comment:
`cmake` should also be a `nativeBuildInput`. Note that `buildPackages.` is not necessary in this case because `mkDerivation` will magically fish it out.